searx/engines/ansa.py

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""Engine for Ansa, Italy's oldest news agency.
+
+To use this engine add the following entry to your engines
+list in ``settings.yml``:
+
+.. code:: yaml
+
+  - name: ansa
+    engine: ansa
+    shortcut: ans
+    disabled: false
+
+"""
+
+from urllib.parse import urlencode
+from lxml import html
+from searx.result_types import EngineResults
+from searx.utils import eval_xpath, eval_xpath_list, extract_text
+
+engine_type = 'online'
+language_support = False
```

Comment:
That doesn't need to be set explicitely, but it also won't hurt :)

Reply:
it that the "default" I guess?
Either way I won't add it again in the future, or if you prefer I will remove it for consistency! :)